Gem/Transfer/ViewsModels/RecipientViewModel.swift

```diff
@@ -19,11 +22,17 @@ enum RecipientScanResult {
     case transferData(TransferData)
 }
 
-class RecipientViewModel: ObservableObject {
+@Observable
+class RecipientViewModel {
     let wallet: Wallet
     let asset: Asset
     let type: RecipientAssetType
-    
+    var address: String = ""
+    var memo: String = ""
+    var amount: String = ""
+    var isPresentingContactList = false
```

Comment:
```suggestion
    var isPresentingContactList = false
```

view model should not be responsible for showing contact list